speakers/kristian-glass.md

```diff
@@ -0,0 +1,3 @@
+name: Kristian Glass
+---
+Etc
```

Comment:
> If you submit more than one proposal, you don't need to send all of these details again.

I followed this advice (from the CfP form). I can rewrite a bio, but using the actual one I believe I submitted would be cool...!